src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue

```diff
@@ -36,6 +36,7 @@ import _filter from 'lodash/filter';
 import _get from 'lodash/get';
 
 import featuredLoansQuery from '@/graphql/query/featuredLoansData.graphql';
+// import categoryServiceRowsQuery from '@/graphql/query/lendByCategory/categoryServiceLoanChannels.graphql';
```

Comment:
Remove?

Reply:
Just getting started on the hero section which will also use this query.